pymfe/mfe.py

```diff
@@ -304,8 +305,10 @@ def _call_summary_methods(
             user must suppress these warnings using some built-in argument of
             the summary method using the kwargs argument, if possible.
 
-        verbose : :obj:`bool`, optional
-            If True, then messages about the summarization process may be
+        verbose : :obj:`int`, optional
+            Select the verbosity level of the summarization process.
```

Comment:
It will be good if verbose=0 do not show anything. I think that this behavior is not present. Could you added it?

Reply:
Oops, sorry. I've completely overlooked this. Now it's (hopefully) fixed.